### PR TITLE
feat(onboard/install): update Sass requirements on dev documentation

### PR DESCRIPTION
**Pull request by tiloyi**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

Update Sass requirements on dev documentation

GitHub issue number or Jira issue URL: [MZC-104](https://jira.adeo.com/browse/MZC-104)

### CODE REVIEW

**Review by caqueste — CHANGES_REQUESTED** [review]

File: src/docs/GetStarted/Developers/Installation/index.mdx [hunk 1]
```diff
@@ -1,37 +1,73 @@
...
 
-we also packaged gatsby extensions that we developed in order to create the styleguide website
+We also packaged [Gatsby](https://www.gatsbyjs.com/) extensions that we developed in order to create the styleguide website
 
-- `@mozaic-ds/gatsby-github-release`: a gatsby plugin that helps us get the older versions of the design system docs
-- `@mozaic-ds/source-directory-tree`: a gatsby plugin to get the directory and file structure in order to build the styleguide menu
-- `@mozaic-ds/source-pattern`: a gatsby plugin build the live examples displayed in the styleguide
-- `@mozaic-ds/theme-styleguide`: the gatsby theme with the components and configuration used to build the styleguide UI
+- `@mozaic-ds/gatsby-github-release`: a Gatsby plugin that helps us get the older versions of the design system docs
+- `@mozaic-ds/source-directory-tree`: a Gatsby plugin to get the directory and file structure in order to build the styleguide menu
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply src/docs/GetStarted/Developers/Installation/index.mdx]: fixed

File: src/docs/GetStarted/Developers/Installation/index.mdx [hunk 2]
```diff
@@ -1,37 +1,73 @@
...
+- `@mozaic-ds/source-pattern`: a Gatsby plugin build the live examples displayed in the styleguide
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply src/docs/GetStarted/Developers/Installation/index.mdx]: fixed

File: src/docs/GetStarted/Developers/Installation/index.mdx [hunk 3]
```diff
@@ -1,37 +1,73 @@
...
+- `@mozaic-ds/theme-styleguide`: the Gatsby theme with the components and configuration used to build the styleguide UI
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply src/docs/GetStarted/Developers/Installation/index.mdx]: fixed

File: src/docs/GetStarted/Developers/Installation/index.mdx [hunk 4]
```diff
@@ -1,37 +1,73 @@
...
+
+Indeed, the package `@mozaic-ds/tokens` is a dependency of the package `@mozaic-ds/styles`.
+
+</Highlight>
 
-#### Others:
+## Others
 
-we also packaged gatsby extensions that we developed in order to create the styleguide website
+We also packaged [Gatsby](https://www.gatsbyjs.com/) extensions that we developed in order to create the styleguide website
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply src/docs/GetStarted/Developers/Installation/index.mdx]: fixed

File: src/docs/GetStarted/Developers/Installation/index.mdx [hunk 5]
```diff
@@ -1,37 +1,73 @@
...
 
-- `@mozaic-ds/css-dev-tools`: tools for css like PostCSS plugins and linters config
 - `@mozaic-ds/styles`: source `.scss` files
 - `@mozaic-ds/tokens`: tokens json sources as well as compiled ones
-- `@mozaic-ds/sketch-libraries`: the sketch designs files
-- `@mozaic-ds/web-fonts`: the webfonts used by mozaic
+- `@mozaic-ds/web-fonts`: the webfonts used by Mozaic
 - `@mozaic-ds/icons`: the icon svg files library
+- `@mozaic-ds/css-dev-tools`: tools for css like PostCSS plugins and linters config
+- `@mozaic-ds/sketch-libraries`: the sketch designs files
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply src/docs/GetStarted/Developers/Installation/index.mdx]: fixed

**Review by mohamedMok — APPROVED** [review]

**Review by caqueste — APPROVED** [review]

**Pull request merged**